ckanext/gcnotify/mailer.py

```diff
@@ -153,6 +153,21 @@ def notify_ckan_user_create(email,
   )
 
 
+def notify_lockout(user, lockout_timeout):
+   # type: (ckan.model.User,int) -> None
+
+    if not user.email:
+      raise mailer.MailerException(_("No recipient email address available!"))
+
+    send_email(
+      recipient=user.email,
+      template_id=get_template_id("notify_lockout"),
+      personalisation={
+        "timeout": int(lockout_timeout / 60)
+      }
+    )
```

Comment:
Would this fit better in the theme? Unless I'm missing something it assumes we're using our ckanext-security customizations and doesn't really look like it belongs here

Reply:
@wardi this extension already monkey patches the mailer with `notify_ckan_user_create` and `send_username_recovery` here https://github.com/open-data/ckanext-gcnotify/blob/master/ckanext/gcnotify/plugin.py#L22

So even if any others use this extension, these monkey patches really don't do anything. They just fit into GC Notify cuz of the get GC Notify template stuff it does in here.

Reply:
`mailer.notify_lockout` isn't an existing function getting monkey-patched so it's pretty weird to be creating it from here, why not create it where it's called to reduce confusion?

Reply:
@wardi `notify_ckan_user_create` and `send_username_recovery` aren't existing functionality either, but all of them have inline comments referring to the GC Notify plugin when we use them in the Canada theme.

And there is an inline comment in our security fork where `notify_lockout` is used as well.

I am waiting for the `INotify` implement from upstream so we can actually make this good.

Reply:
ok, the acknowledgement that this is terrible but at least it's the same terrible as the other terrible things we're already planning to replace, is good enough.